scripts/install/opt/seagate/cortx/motr/bin/motr_setup

```diff
@@ -356,5 +358,5 @@ def main(argv: dict):
         return errno.EINVAL
 
 if __name__ == '__main__':
+    signal.signal(signal.SIGTERM, receiveSigTerm)
     sys.exit(main(sys.argv))
```

Comment:
How SIGKILL is handled here ?

Reply:
python cannot handle SIGKILL signal. 
https://docs.python.org/3/library/signal.html#signal.SIGKILL)